pkg/collector/corechecks/net/wlan/wlan_darwin.m

```diff
@@ -20,8 +20,8 @@ - (instancetype)init {
     if (self) {
         self.locationManager = [[CLLocationManager alloc] init];
 
-        if (@available(macOS 10.15, *)) {
-            CLAuthorizationStatus status = [CLLocationManager authorizationStatus];
+        if (@available(macOS 11.0, *)) {
+            CLAuthorizationStatus status = [self.locationManager authorizationStatus];
```

Comment:
```suggestion
            CLAuthorizationStatus status = self.locationManager.authorizationStatus;
```

https://developer.apple.com/documentation/corelocation/cllocationmanager/authorizationstatus-swift.property?language=objc

`authorizationStatus` seems to have become a instance property, we can access it directly.